citesphere/src/main/webapp/WEB-INF/views/auth/group/item.html

```diff
@@ -27,38 +27,24 @@
             
 			$("#parsed-ref-modal").modal();
 		});
-		
 	});
 	
-	$(document).ready(function(){
-		$.ajax({ 
-			'url': '[(${citation.key})]/paging?index=[(${index})]&page=[(${page})]&sortBy=[(${sortBy})]&collectionId=[(${collectionId})]',
-			'type': "GET",
-			'success': function(response){
-				if(response.next != null){
-					document.getElementById("next").addEventListener("click", function(e) {
-						window.location.href = response.next + "?index=" + response.nextIndex + "&page=" + response.nextPage + "&sortBy=" + response.sortBy + "&collectionId=" + response.collectionId;
-					});
-					$('#next').attr("disabled", false);
-					$('#next').removeClass("btn btn-default").addClass("btn btn-primary");
-				}
-				
-				if(response.prev != null){
-					document.getElementById("prev").addEventListener("click", function(e) {
-						window.location.href = response.prev + "?index=" + response.prevIndex + "&page=" + response.prevPage + "&sortBy=" + response.sortBy + "&collectionId=" + response.collectionId;
-					});
-					$('#prev').attr("disabled", false);
-					$('#prev').removeClass("btn btn-default").addClass("btn btn-primary");
-				}
-				$("#nextSpinner").hide(),
-				$("#prevSpinner").hide();
-			},
-			'error': function(data) {
-				$("#nextSpinner").hide(),
-				$("#prevSpinner").hide();
-			}
-		});
-		
+	$(document).ready(function() {
+	    var queryParams = window.location.search;
+	    if(queryParams)  {
+	        const params = new URLSearchParams(queryParams);
+	        params.delete('page');
+	        params.delete('searchTerm');
+	        params.delete('sortBy');
+	        params.delete('index');
+	        params.delete('collectionId');
+	        queryParams = params.toString();
+	    }
+	    
+	    $('.btn-back').attr("href", $('.btn-back').attr("href") + "&" + queryParams);
+	    $('.btn-prev').attr("href", $('.btn-prev').attr("href") + "&" + queryParams);
+	    $('.btn-next').attr("href", $('.btn-next').attr("href") + "&" + queryParams);
```

Comment:
since these three lines are pretty much the same, just give them all the same class and have only one line with the class selelctor.